odxtools/tablerow.py

```diff
@@ -25,8 +25,12 @@ class TableRow(IdentifiableElement):
     key_raw: str
     structure_ref: Optional[OdxLinkRef]
     structure_snref: Optional[str]
+
+    # the referenced DOP must be a simple DOP (i.e.,
+    # DataObjectProperty, cf section 7.3.6.11 of the spec)!
```

Comment:
A DtcDop is also a simple DOP

Reply:
good point. fixed: c6dbcdad092ad43f14ce69ae6fb4f63284b5b692